Batchbald_implementation.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    latexify(width_scale_factor=2, fig_height=2)
</pre><p>Can you adjust this as the following logic?</p><p> </p><p>If a figure is covering 1/3 of the page width, we should set width_scale_factor=3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
It is set to 2 because it generates only 2 figures. The accuracy plot and the visualisation for the digits picked.